### DISCUSSION TITLE
I2C dual motor driver shield based on HR8833

### DISCUSSION THREAD

**Discussion by codystoica** [arendst/Tasmota#16888] [Q&A]
Hello
I tried to connect LOLIN I2C dual motor driver shield based on HR8833 to D1 mini but it does not recognize it as a device
When I run I2Cscan, a device at address 0x30 is detected, but it does not recognize it. Can you help me with this problem, what could I do?
https://github.com/wemos/LOLIN_I2C_MOTOR_Library

https://fr.aliexpress.com/item/1005003697301523.html?spm=a2g0o.order_list.0.0.21ef5e5bEnFCsf&gatewayAdapt=glo2fra

https://www.wemos.cc/en/latest/d1_mini_shield/hr8833_motor.html

**Comment by barbudor** [comment]
As already responsed on the Motor driver discussion, I oly see in the code the driver for the shield V1 with TB6612FNG

**Reply by vic42** [reply]
Please have a look at my answer at https://github.com/arendst/Tasmota/discussions/10387.
Bottom line: You have to compile your own build including appropriate settings.
